tests/bases/test_collections.py

```diff
@@ -168,6 +168,25 @@ def test_metric_collection_prefix_postfix_args(prefix, postfix):
     for name in names:
         assert f"new_prefix_{name}" in out, 'prefix argument not working as intended with clone method'
 
+    for k, _ in new_metric_collection.items():
+        assert 'new_prefix_' in k
+
+    for k in new_metric_collection.keys():
+        assert 'new_prefix_' in k
+
+    for k, _ in new_metric_collection.items(keep_base=True):
+        assert 'new_prefix_' not in k
+
+    for k in new_metric_collection.keys(keep_base=True):
+        assert 'new_prefix_' not in k
+
+    repr = 'MetricCollection(' \
+           '\n  prefix=new_prefix_,' \
+           f'\n  postfix={postfix},' \
+           '\n  (DummyMetricSum): DummyMetricSum()' \
+           '\n  (DummyMetricDiff): DummyMetricDiff()\n)'
```

Comment:
I think this would look nicer if prefix/postfix were at the end

Reply:
I am using `extra_repr` hook and this adds to the debugging by default.

Reply:
Done !